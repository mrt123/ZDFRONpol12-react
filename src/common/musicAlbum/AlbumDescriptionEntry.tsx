const descriptionEntryStyle = {
  margin: "5px",
  fontSize: "11px",
};

interface AlbumDescriptionEntryProps {
  value: string | number | null;
}

const AlbumDescriptionEntry = ({ value }: AlbumDescriptionEntryProps) => {
  if (value === null) return null;

  return <div style={descriptionEntryStyle}>{value}</div>;
};

export default AlbumDescriptionEntry;
