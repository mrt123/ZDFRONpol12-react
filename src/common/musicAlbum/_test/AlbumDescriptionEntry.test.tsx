import { render } from "@testing-library/react";
import AlbumDescriptionEntry from "../AlbumDescriptionEntry";

test("AlbumDescriptionEntry", () => {
  const { container } = render(
    <AlbumDescriptionEntry value="testowy item albumu" />
  );

  expect(container).toMatchSnapshot();
});
