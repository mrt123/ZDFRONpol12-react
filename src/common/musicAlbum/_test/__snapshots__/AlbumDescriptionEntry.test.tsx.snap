// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlbumDescriptionEntry 1`] = `
<div>
  <div
    style="margin: 5px; font-size: 11px;"
  >
    testowy item albumu
  </div>
</div>
`;
